devolving_music/lib/elo_scoring.py

```diff
@@ -0,0 +1,41 @@
+# Python 3 program for Elo Rating
+import math
+
+# Function to calculate the Probability
+def Probability(rating1, rating2):
```

Comment:
```suggestion
def probability(rating1, rating2):
```

All functions in this file should be snake case.